### ISSUE TITLE
Auto-Fetch does not work

### ISSUE THREAD

**Issue by aaronols** [zielu/GitToolBox#183]
I have a multi-repo multi-module IntelliJ project.
At some point in the last weeks, the auto-fetch functionality stopped working for me.
I noticed this because deleted remote branches were no longer pruned from my local clones.
Also, auto-fetching would always leave a "Fetch Successful" message in the Event Log which no longer happens.

Manually triggering the fetch process via VCS -> Git -> Fetch menu entry still works as expected.

Some info about my set up:
```
IntelliJ IDEA 2019.3 EAP (Ultimate Edition)
Build #IU-193.5233.12, built on November 13, 2019
Runtime version: 11.0.4+10-b520.6 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Registry: search.everywhere.settings=true, ide.balloon.shadow.size=0
Non-Bundled Plugins: com.intellij.plugins.watcher, BashSupport, GrepConsole, Lombook Plugin, Pythonid, com.dmarcotte.handlebars, com.jetbrains.plugins.meteor, com.perl5, izhangzhihao.rainbow.brackets, name.kropp.intellij.makefile, org.jetbrains.plugins.vue, org.igu.plugins.nativescript, org.majki.intellij.ldapbrowser, org.nik.presentation-assistant, org.sonarlint.idea, some.awesome, tanvd.grazi, zielu.gittoolbox
```
GitToolBox version: `193.1.0`

I did find the following log lines multiple times throughout the idea.log file:
```
WARN - com.intellij.util.xmlb.Binding - no accessors for class zielu.gittoolbox.config.ReferencePointForStatusConfig 
INFO - .fetch.AutoFetchOnBranchSwitch - Auto-fetch delay on branch switch is PT0S 
INFO - lbox.fetch.AutoFetchSubscriber - Branch switch not eligible for auto-fetch: previous=RepoInfo[.......]
```
There are however no exceptions in the log.

I have Auto-fetch enabled and set to 15min interval as well as on branch switch.
Parent branch mode is auto and there are no auto fetch exclusions.

I have not yet tried to replicate this with a new (empty) project or new (empty) IntelliJ profile/settings.

My apologies for the bare-bones error report. I don't really now how to start zeroing in on this problem.
What steps would help you to analyze why auto-fetching is not triggered anymore with my IntelliJ setup?

**Comment by zielu** [comment]
@aaronols actually I recently started having suspicions that it was broken at same point of time. I'll dig into this asap

**Comment by zielu** [comment]
@aaronols I found condition that was reversed when migrating to kotlin. Instead of preventing fetch during build it allowed it only during building. I'll release fix today.

**Comment by zielu** [comment]
Fixed in 193.1.1

**Comment by aaronols** [comment]
Can confirm that auto-fetch works again. Thank you very much for looking into this so quickly.
